packages/node_modules/@ciscospark/internal-plugin-conversation/src/conversation.js

```diff
@@ -994,6 +994,36 @@ const Conversation = SparkPlugin.extend({
   };
 });
 
+/**
+ * Sets/unsets space property for convo
+ * @param {Object} conversation
+ * @param {string} tag
+ * @param {Activity} activity
+ * @returns {Promise<Activity>}
+ */
+[
+  `setSpaceProperty`,
+  `unsetSpaceProperty`
+].forEach((fnName) => {
+  Conversation.prototype[fnName] = function submitObjectActivity(conversation, tag, activity) {
+    if (!isString(tag)) {
+      return Promise.reject(new Error(`\`tag\` must be a string`));
+    }
+
+    const verb = fnName.startsWith(`set`) ? `set` : `unset`;
```

Comment:
Move this outside of `submitObjectActivity` so that it only does this check twice instead of everytime this function gets called.